doc/api/esm.md

```diff
@@ -299,6 +299,8 @@ The `type: 'json'` attribute is mandatory when importing JSON modules.
 
 ## Built-in modules
 
+> **Note:** When importing built-in modules via ESM , the module is fully evaluated at import time, and all properties are populated immediately. This can make initial imports slightly slower compared to CommonJS, which evaluates lazily.
```

Comment:
> This can make initial imports slightly slower compared to CommonJS, which evaluates lazily.

This is not true. The builtin module body is still been evaluated fully at the time being imported in CommonJS. However, some exported names are lazily initialized when they are accessed at the time, which could make the comparable differences.

Please see original post for accurate reference: https://github.com/nodejs/node/issues/59686#issuecomment-3242590976